src/Microsoft.OData.Core/ODataBatchReader.cs

```diff
@@ -745,7 +484,7 @@ private void VerifyCallAllowed(bool synchronousCall)
         /// <summary>
         /// Increases the size of the current batch message; throws if the allowed limit is exceeded.
         /// </summary>
-        private void IncreaseBatchSize()
+        internal void IncreaseBatchSize()

```

Comment:
>IncreaseBatchSize [](start = 22, length = 17)

It looks like this is only called for MultipartMixed; shouldn't it also be called from the JSON format? Should it be protected?